src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -0,0 +1,162 @@
+package de.zalando.aruha.nakadi.repository.db;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.zalando.aruha.nakadi.NakadiException;
+import de.zalando.aruha.nakadi.config.NakadiConfig;
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.EventTypeSchema;
+import de.zalando.aruha.nakadi.repository.DuplicatedEventTypeNameException;
+import de.zalando.aruha.nakadi.repository.EventTypeRepository;
+import de.zalando.aruha.nakadi.repository.NoSuchEventTypeException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.jdbc.support.rowset.SqlRowSet;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNull.notNullValue;
+
+public class EventTypeDbRepositoryTest {
```

Comment:
I would like to run these tests without having to wait for the full application to boot. In order to do so, we created another ticket https://techjira.zalando.net/browse/ARUHA-86 which we'll be solved after ARUHA-50.